backend/danswer/auth/invited_users.py

```diff
@@ -0,0 +1,21 @@
+from typing import cast
+
+from danswer.dynamic_configs.factory import get_dynamic_config_store
+from danswer.dynamic_configs.interface import ConfigNotFoundError
+from danswer.dynamic_configs.interface import JSON_ro
+
+USER_STORE_KEY = "USERS"
```

Comment:
nit: might be more descriptive to call this something like `INVITED_USERS` just so if someone is looking through the DB they can easily tell what this is

Reply:
Done: ed0c8533